### PR TITLE
Fix AOCODAF405V3 motor5/6/7/8 issue 

### PR DESCRIPTION
**Pull request by ChenYang78256**
Add motor 5 ，motor 6，motor7，motor 8 configuration

### CODE REVIEW

**Comment by ChenYang78256** [comment]
After testing, bf turned on ESC_SENSOR could not display the motor speed, the reason has not been found

**Comment by lida2003** [comment]
> After testing, bf turned on ESC_SENSOR could not display the motor speed, the reason has not been found

[ ESC telemetry RPM lost in 4.5 firmware #13283 ](https://github.com/betaflight/betaflight/issues/13283)

@YangChen1234 Please keep track of #13283 issue and support if any progress.

**Comment by ChenYang78256** [comment]
In addition to the problem of ESC telemetry RPM lost, other functions of F405V3 Flash V3 self test passed.

File: configs/AOCODAF405V3/config.h [hunk 1]
```diff
@@ -49,6 +49,11 @@
 #define MOTOR2_PIN PC7
 #define MOTOR3_PIN PC8
 #define MOTOR4_PIN PC9
+#define MOTOR5_PIN PA15
+#define MOTOR6_PIN PA8
+#define MOTOR7_PIN PB10
+#define MOTOR8_PIN PB11
+#define RX_PPM_PIN PA3
```
Comment (lida2003) [inline configs/AOCODAF405V3/config.h:56]: What is the usage of RX_PPM_PIN ?

**Comment by ChenYang78256** [comment]
I found that AOCODAF405V2MPU6000 in our company's library has this configuration, which can use ppm receivers, but our company does not have such receivers at present

**Comment by ChenYang78256** [comment]
I found that AOCODAF405V2MPU6000 in our company's library has this configuration, which can use ppm receivers, but our company does not have such receivers at present
&nbsp;


～～～
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Daniel ***@***.***&gt;; 
发送时间: 2024年1月10日(星期三) 晚上9:01
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [Aocoda-RC/betaflight-config] Update AOCODAF405V3 config.h (PR #7)





 
@lida2003 commented on this pull request.
 
 
In configs/AOCODAF405V3/config.h:
 &gt; @@ -49,6 +49,11 @@  #define MOTOR2_PIN PC7  #define MOTOR3_PIN PC8  #define MOTOR4_PIN PC9 +#define MOTOR5_PIN PA15 +#define MOTOR6_PIN PA8 +#define MOTOR7_PIN PB10 +#define MOTOR8_PIN PB11 +#define RX_PPM_PIN PA3  
What is the usage of RX_PPM_PIN ?

**Comment by lida2003** [comment]
> I found that AOCODAF405V2MPU6000 in our company's library has this configuration, which can use ppm receivers, but our company does not have such receivers at present

OK, thanks!

**Pull request merged**